src/app/modules/auth-private/sign-up/sign-up.component.ts

```diff
@@ -1,15 +1,17 @@
 import { Component, OnInit, EventEmitter, ViewChild } from '@angular/core';
 import { Apollo } from 'apollo-angular';
-import { SignUpMutation, CitiesSearchQuery, LocationListQuery } from './sign-up.graphql';
+import { SignUpMutation, CitiesSearchQuery, LocationListQuery, ConsultationDeadlineQuery, AuhtAcceptInviteMutation } from './sign-up.graphql';
```

Comment:
@Nishanthh Based on our conversation in the call, I think `ConsultationDeadlineQuery` can be removed, right?

Reply:
@avikt removed bro